apis/python/src/tiledbsoma/__init__.py

```diff
@@ -19,24 +20,25 @@
 __version__ = get_implementation_version()
 
 __all__ = [
-    "get_implementation",
-    "get_implementation_version",
-    "get_SOMA_version",
-    "get_storage_engine",
-    "show_package_versions",
-    "stats_enable",
-    "stats_disable",
-    "stats_reset",
-    "stats_dump",
     "AxisColumnNames",
     "AxisQuery",
-    "ExperimentAxisQuery",
     "Collection",
+    "DataFrame",
     "DenseNDArray",
     "DoesNotExistError",
     "Experiment",
-    "SOMAError",
-    "DataFrame",
+    "ExperimentAxisQuery",
+    "get_implementation_version",
+    "get_implementation",
+    "get_SOMA_version",
+    "get_storage_engine",
     "Measurement",
+    "open",
+    "show_package_versions",
+    "SOMAError",
     "SparseNDArray",
+    "stats_disable",
```

Comment:
I missed the PR that added these -- IMHO they should not be at the top-level namespace, as they are TileDB-specific.

Reply:
This is not part of the current change, but I disagree on this assessment—I think it makes sense for `tiledbsoma` to directly export additional functionality *in addition to* the basic SOMA functionality. If somebody is importing `tiledbsoma`, they already are binding their implementation to `tiledbsoma` specifically.

Reply:
I do not feel strongly about this, so I leave it to a consensus decision....

Reply:
Agreed with @thetorpedodog that this is unrelated to this PR. The work was done on #865 and #871. Putting stats routines at top level was worked out in the PRs, but indeed could (and should) have been discussed more broadly than on the PRs.

I've added this to the agenda for our Friday spec-discussion meeting, and have also upated issue #872.